coalib/results/Diff.py

```diff
@@ -173,18 +173,40 @@ def affected_code(self, filename):
         :param filename: The filename to associate the SourceRange's to.
         :return:         A list of all related SourceRange objects.
         """
-        return list(diff.range(filename) for diff in self.split_diff())
+        return list(diff.range(filename)
+                    for diff in self.split_diff(distance=0))
 
-    def split_diff(self):
+    def split_diff(self, distance=1):
```

Comment:
having one unchanged line in between makes a bit of sense since those changes likely still belong together. This is about splitting semantical changes after all!